### PR TITLE
Fix IPv6 functionality

### PR DESCRIPTION
**Pull request by middleagedman**
Added support for some unreferenced variables needed in bash_completion
Added php5-sockets to apk install
Updated README examples
IPv6 address shows correct on the admin interface.

### CODE REVIEW

File: alpine.docker [hunk 1]
```diff
@@ -18,7 +18,7 @@ RUN apk add --update 'gnupg<2.1.17-r0' && \
     gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 0527A9B7 && \
     gpg --verify /tini.asc && \
     chmod +x /tini && \
-    apk add wget bash && \
```
Comment (diginc) [inline alpine.docker]: Can you explain what php5-sockets fixes?  I'm not seeing the equivilent installed in pi-hole/pi-hole automated installer.

**Comment by middleagedman** [comment]
Cool whatever works. Yea the php5-sockets gets rid of the error message on the ipv6 box that says "your php was not compiled with sockets support"  it's only applicable if you have ipv6 turned on.

**Comment by diginc** [comment]
@middleagedman can you take a look at https://github.com/diginc/docker-pi-hole/pull/123/files and ctrl+find ipv6 references to see if I got everything you had covered?  I moved your dockerfile change to /install.sh where the other php5 installs happen

**Comment by middleagedman** [comment]
Looks good.  I'll try and build and test

**Comment by diginc** [comment]
Dev tags are published https://hub.docker.com/r/diginc/pi-hole/tags/
Closing since your code is in #123

**Pull request closed without merging**